Library/Homebrew/test/cask/audit_spec.rb

```diff
@@ -399,6 +399,28 @@ def include_msg?(messages, msg)
       end
     end
 
+    describe "blacklist checks" do
+      let(:cask_token) { "with-suite" }
```

Comment:
I chose this test Cask as I wanted to use a Cask that would pass the audit successfully. Is there a more generic option than `with-suite`?

Reply:
`local-caffeine` is a bit more generic.